eZ/Publish/Core/FieldType/Image/Value.php

```diff
@@ -177,4 +179,27 @@ public function __set($propertyName, $propertyValue)
 
         throw new PropertyNotFoundException($propertyName, get_class($this));
     }
+
+    public function isEquals(SPIValue $value): bool
+    {
+        return $this->toHash($value) === $this->toHash($this);
```

Comment:
Hows that differ from https://github.com/ezsystems/ezpublish-kernel/pull/3060/files#diff-d9c14037c00e55c36cbbe773820aa561R1753